### PR TITLE
GH-6 Add unit tests for none and blank.

### PR DESCRIPTION
**Pull request by Rollczi**

### CODE REVIEW

**Review by dzikoysk — APPROVED** [review]

File: expressible/src/test/kotlin/panda/std/OptionTest.kt [hunk 1]
```diff
@@ -35,6 +34,22 @@ import kotlin.NoSuchElementException
 
 class OptionTest {
 
+    @Test
+    fun `should not present none`() {
```
Comment (dzikoysk) [inline expressible/src/test/kotlin/panda/std/OptionTest.kt:38]: should be empty

File: expressible/src/test/kotlin/panda/std/OptionTest.kt [hunk 2]
```diff
@@ -35,6 +34,22 @@ import kotlin.NoSuchElementException
...
+        val none = none<Void>()
```
Comment (dzikoysk) [inline expressible/src/test/kotlin/panda/std/OptionTest.kt:39]: Replace Void with String to simplify example

**Review by dzikoysk — CHANGES_REQUESTED** [review]

**Review by dzikoysk — APPROVED** [review]

**Comment by dzikoysk** [comment]
Thanks!
